Formula/postgresql.rb

```diff
@@ -138,35 +138,12 @@ def caveats
     EOS
   end
 
-  plist_options manual: "pg_ctl -D #{HOMEBREW_PREFIX}/var/postgres start"
```

Comment:
I remove this because it seemingly had no effect (the `run` command was still appearing on `brew info postgresql`)

Reply:
It'll be generated if a service is provided.

Reply:
does it matter that the `manual` command here was different? i.e. using `pg_ctl` over `postgres`?

Reply:
No, I think the wording in the output also suggests that it'll be running exactly what the `service` block runs.